Fuzzing/rstp.txt

```diff
@@ -0,0 +1,354 @@
+/
+/0
+/0/video1
+/1
+/1.AMP
+/1/1:1/main
+/1/cif
+/1/stream1
+/11
+/12
+/4
+/CAM_ID.password.mp2
+/CH001.sdp
+/GetData.cgi
+/H264
+/HighResolutionVideo
+/HighResolutionvideo
+/Image.jpg
+/LowResolutionVideo
+/MJPEG.cgi
+/MediaInput/h264
+/MediaInput/h264/stream_1
+/MediaInput/mpeg4
+/ONVIF/MediaInput
+/ONVIF/channel1
+/PSIA/Streaming/channels/0?videoCodecType=H.264
+/PSIA/Streaming/channels/1
+/PSIA/Streaming/channels/1?videoCodecType=MPEG4
+/PSIA/Streaming/channels/h264
+/Possible
+/ROH/channel/11
+/Streaming/Channels/1
+/Streaming/Channels/101
+/Streaming/Channels/102
+/Streaming/Channels/103
+/Streaming/Channels/2
+/Streaming/Unicast/channels/101
+/Streaming/channels/101
+/Video?Codec=MPEG4&Width=720&Height=576&Fps=30
+/VideoInput/1/h264/1
+/access_code
+/access_name_for_stream_1_to_5
+/av0_0
+/av0_1
+/av2
+/avn=2
+/axis-media/media.amp
+/axis-media/media.amp?videocodec=h264&resolution=640x480
+/cam
+/cam/realmonitor
+/cam/realmonitor?channel=1&subtype=00
+/cam/realmonitor?channel=1&subtype=01
+/cam/realmonitor?channel=1&subtype=1
+/cam0_0
+/cam0_1
+/cam1/h264
+/cam1/h264/multicast
+/cam1/mjpeg
+/cam1/mpeg4
+/cam1/onvif-h264
+/camera.stm
+/cgi-bin/viewer/video.jpg?resolution=640x480
+/ch0
+/ch0.h264
+/ch001.sdp
+/ch01.264
+/ch0_0.h264
+/ch0_unicast_firststream
+/ch0_unicast_secondstream
+/channel1
+/dms.jpg
+/dms?nowprofileid=2
+/h264
+/h264.sdp
+/h264/ch1/sub/
+/h264/media.amp
+/h264Preview_01_main
+/h264Preview_01_sub
+/cam4/mpeg4
+/h264_vga.sdp
+/image.jpg
+/image.mpg
+/image/jpeg.cgi
+/img/media.sav
+/img/video.asf
+/img/video.sav
+/ioImage/1
+/ipcam.sdp
+/ipcam/stream.cgi?nowprofileid=2
```

Comment:
I see a lot of URL parameters in this file. It might be better to list them all in the folder readme and replace the parameters with placeholders in the original file. That would increase the coverage this wordlist has on real cameras.